package.json

```diff
@@ -13,28 +13,30 @@
   },
   "dependencies": {
     "aws-sdk": "^2.1.49",
-    "mime": "^1.3.4"
+    "mime": "^2.3.1"
```

Comment:
Can we add the `engines` property with the node version?

Reply:
Added